python/dgl/nn/pytorch/conv/graphconv.py

```diff
@@ -25,6 +155,20 @@ class GraphConv(nn.Module):
     (i.e.,  :math:`c_{ij} = \sqrt{|\mathcal{N}(i)|}\sqrt{|\mathcal{N}(j)|}`),
     and :math:`\sigma` is an activation function.
 
+    If a weight tensor on each edge is provided, the weighted graph convolution is defined as:
+
+    .. math::
+      h_i^{(l+1)} = \sigma(b^{(l)} + \sum_{j\in\mathcal{N}(i)}\frac{e_{ji}}{c_{ij}}h_j^{(l)}W^{(l)})
```

Comment:
It seems that `\frac{e_{ji}}{c_{ij}}` should be `e_{ji} * c_{ij}`.

Reply:
Why is that? `c_{ij}` is defined as `\sqrt{|mathcal{N}(i)|}\sqrt{|mathcal{N}(i)|}`, without `^{-1}`.

Reply:
Never mind. Got lost across notations :)